2025-02-25 20:30:23.723 [info] 




2025-02-25 20:30:23.747 [info] Extension host agent started.
2025-02-25 20:30:23.927 [info] [<unknown>][58c6168e][ExtensionHostConnection] New connection established.
2025-02-25 20:30:23.939 [info] [<unknown>][d5b565ab][ManagementConnection] New connection established.
2025-02-25 20:30:24.306 [info] [<unknown>][58c6168e][ExtensionHostConnection] <98529> Launched Extension Host Process.
2025-02-25 20:30:28.220 [error] CodeExpectedError: Could not find pty 5 on pty host
    at I.W (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12233)
    at I.updateTitle (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at Kc.s (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80170)
    at Kc.q (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79693)
    at hs.value (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79095)
    at C.B (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at C.fire (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.w (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29136)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-02-25 20:30:28.230 [error] CodeExpectedError: Could not find pty 5 on pty host
    at I.W (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12233)
    at I.updateIcon (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7772)
    at O.s.<computed> (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at Kc.s (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80170)
    at Kc.q (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79693)
    at hs.value (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79095)
    at C.B (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at C.fire (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.w (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29136)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-02-25 20:30:44.388 [error] [File Watcher] Unexpected error: inotify_add_watch on '/root/.vscode-server/data/logs/20240416T060052/exthost1/vscode.github' failed: No such file or directory (EUNKNOWN) (path: /root)
2025-02-25 20:30:44.389 [error] [File Watcher (universal)] inotify_add_watch on '/root/.vscode-server/data/logs/20240416T060052/exthost1/vscode.github' failed: No such file or directory
2025-02-25 20:35:23.750 [info] New EH opened, aborting shutdown
2025-02-26 01:37:25.985 [info] [<unknown>][d5b565ab][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
2025-02-26 01:37:26.180 [info] [<unknown>][58c6168e][ExtensionHostConnection] <98529> Extension Host Process exited with code: 0, signal: null.
2025-02-26 01:37:26.180 [info] Last EH closed, waiting before shutting down
2025-02-26 01:42:26.183 [info] Last EH closed, shutting down
